packages/stone/stone.0.3.4/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "armael@isomorphis.me"
+author: "armael@isomorphis.me"
+homepage: "http://dev.isomorphis.me/stone"
+bug-reports: "http://github.com/Armael/stone/issues"
+dev-repo: "http://github.com/Armael/stone"
```

Comment:
You mean for `dev-repo`? If so, I pushed a new version with `.git` added at the end of the url.